src/app/WriteClient.cpp

```diff
@@ -270,7 +270,7 @@ CHIP_ERROR WriteClient::SendWriteRequest(NodeId aNodeId, Transport::AdminId aAdm
 
     err = mpExchangeCtx->SendMessage(
         Protocols::InteractionModel::MsgType::WriteRequest, std::move(packet),
-        Messaging::SendFlags(Messaging::SendMessageFlags::kExpectResponse).Set(Messaging::SendMessageFlags::kNoAutoRequestAck));
+        Messaging::SendFlags(Messaging::SendMessageFlags::kExpectResponse));
```

Comment:
This doesn't appear to be a typo? What bug is fixed here, what new test was added or modified to confirm this behavior?

Reply:
This is MRP kNoAutoRequestAck flag, basically it means disable MRP here, we previously disable MRP in Command/Read for stability reason, I think song and Boris re-enable MRP for Command/Read, it doesn't impact the happy test behavior. I intend to enable MRP for write, but somehow I forget to remove this kNoAutoRequestAck.
This is similar to what we have done for Command/Read, I have validated it manually and locally to check if MRP is enabled.

Reply:
this has been handled by https://github.com/project-chip/connectedhomeip/pull/7991, removed this from this pr